File: python/mprpc/rpc/address.py
class Address:
    """
    The class to represent the RPC address.
    Currently, only IPV4 is supported in this version.
    
    TODO(kzk): support IPV6, and UnixDomainSocket
    """

    def __init__(self, host, port):
        self._host = host
        self._port = port

    def get_host(self):
        return self._host

    def get_port(self):
        return self._port

File: php/MessagePackRPC/Address.php
<?php
class MessagePackRPC_Address
{
  public $host = null;
  public $port = null;

  public function __construct($host, $port)
  {
    $this->host = $host;
    $this->port = $port;
  }

  public function getHost()
  {
    return $this->host;
  }

  public function getPort()
  {
    return $this->port;
  }
}